chatmm/src/main/java/com/likeminds/chatmm/chatroom/detail/view/adapter/databinder/LMChatAttachmentItemViewDataBinder.kt

```diff
@@ -0,0 +1,48 @@
+package com.likeminds.chatmm.chatroom.detail.view.adapter.databinder
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.likeminds.chatmm.chatroom.detail.model.LMChatAttachmentItemViewData
+import com.likeminds.chatmm.chatroom.detail.view.adapter.LMChatAttachmentBarAdapterListener
+import com.likeminds.chatmm.databinding.LmChatItemAttachmentBinding
+import com.likeminds.chatmm.utils.customview.ViewDataBinder
+import com.likeminds.chatmm.utils.model.ITEM_ATTACHMENT_ITEM
+
+class LMChatAttachmentItemViewDataBinder(
```

Comment:
rename this class to `LMChatAttachmentBarItemViewDataBinder` or `LMChatAttachmentPickerItemViewDataBinder` and accordingly rename the layout file